Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -55,8 +55,10 @@ public protocol AppDelegateViewModelInputs {
   /// Call when the user taps "OK" from the contextual alert.
   func didAcceptReceivingRemoteNotifications()
 
-  /// Call with the result of evaluating Qualtrics Targeting Logic
-  func didEvaluateQualtricsTargetingLogic(with result: QualtricsResultType)
+  /// Call with the result of evaluating Qualtrics Targeting Logic and current Qualtrics.Properties
+  func didEvaluateQualtricsTargetingLogic(
+    with result: QualtricsResultType, properties: QualtricsPropertiesType
```

Comment:
Can we put the first argument on the same line as the definition 😄 

```
func didEvaluateQualtricsTargetingLogic(with result: QualtricsResultType, 
                                        properties: QualtricsPropertiesType)
```

Reply:
I can do:

```swift
  public func didEvaluateQualtricsTargetingLogic(
    with result: QualtricsResultType,
    properties: QualtricsPropertiesType
  ) {
```